tests/php/Extension/MemberExtensionTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace SilverStripe\MFA\Tests\Extension;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+class MemberExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'MemberExtensionTest.yml';
+
+    public function testAdminUserCanViewButNotEditOthersMFAConfig()
+    {
+        $adminMember = $this->objFromFixture(Member::class, 'admin');
+        $targetMember = $this->objFromFixture(Member::class, 'squib');
+
+        Security::setCurrentUser($adminMember);
```

Comment:
FYI you can also do `$this->logInAs($adminMember)`, or even `$this->logInAs('admin')` (where `admin` is the fixture name)